samples/diagnostics/program-visualization/TraceExample.ipynb

```diff
@@ -0,0 +1,555 @@
+       "</code></pre>\n",
+       "<h5>Example</h5><p>Visualize operations at depth 2 on the execution path of a Q# operation defined\n",
+       "as <code>operation MyOperation() : Result</code>:</p>\n",
+       "<pre><code>In []: %trace MyOperation --depth=2\n",
+       "Out[]: &lt;HTML visualization of the operation&gt;\n",
+       "</code></pre>\n"
+      ],
+      "text/plain": [
+       "%trace:\n",
+       "Visualizes the execution path of the given operation."
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "%trace?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "### Basic operations\n",
+    "To start off, let's see how we can debug basic operations with the `%trace` magic command. First, let's define an operation `RunBasicOp` that will perform an `H` on the first qubit and an `X` on the second."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "application/json": "[\"RunBasicOp\"]",
+      "text/html": [
+       "<ul><li>RunBasicOp</li></ul>"
+      ],
+      "text/plain": [
+       "RunBasicOp"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "operation RunBasicOp() : Unit {\n",
+    "    using (qs = Qubit[2]) {\n",
+    "        H(qs[0]);\n",
+    "        X(qs[1]);\n",
+    "        ResetAll(qs);\n",
+    "    }\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "Now, we use the `%trace` magic command to render a visualization of `RunBasicOp` in the cell."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "application/json": "{\"Html\":\"<div id='execution-path-container-bc59252f-4429-4e53-87e5-d06a66976ef9' />\"}",
+      "text/html": [
+       "<div id='execution-path-container-bc59252f-4429-4e53-87e5-d06a66976ef9' />"
+      ],
+      "text/plain": [
+       "Microsoft.Quantum.IQSharp.Jupyter.DisplayableHtmlElement"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "%trace RunBasicOp"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "We can extend this to visualize useful quantum algorithms, such as the Deutsch–Jozsa algorithm. The following operation can be found in our Q# samples [here](https://github.com/microsoft/Quantum/blob/master/samples/getting-started/simple-algorithms/SimpleAlgorithms.qs)."
```

Comment:
This link needs to be changed to: https://github.com/microsoft/Quantum/blob/main/samples/getting-started/simple-algorithms/DeutschJozsa.qs